### PR TITLE
Aaron troska

### PR DESCRIPTION
**Pull request by Trosaar**
pull for stand-up report MVP done.

### CODE REVIEW

File: Project-II-Calculator/calculator/src/App.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  font-size: 5vw;
+}
+
+.work {
+  width: 100%;
+  display: flex;
+  flex-direction: row-reverse;
```
Comment (ari-mercado) [inline Project-II-Calculator/calculator/src/App.css:13]: Impressive use of *row-reverse*!

File: Project-II-Calculator/calculator/src/App.js [hunk 2]
```diff
@@ -1,21 +1,34 @@
...
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
+    <div className='container'>
+      <CalculatorDisplay />
+      <div className='work'>
+        <div className='work-buttons'>
+          {mathSymbols.map(symbol => {
```
Comment (ari-mercado) [inline Project-II-Calculator/calculator/src/App.js:26]: A unique key should be used for each element in the list.
https://reactjs.org/docs/lists-and-keys.html

File: Project-II-Calculator/calculator/src/components/ButtonComponents/ActionButton.js [hunk 3]
```diff
@@ -1,2 +1,43 @@
...
+const ActionButton = () => {
+  return (
+    <div className='white'>
+      <button>
+        clear
+      </button>
+      <div className='numb-buttons'>
+      {numButtons.map(num => {
+        return <NumberButton text={num} buttonStyle='buttonStyle'/>
+      })}
```
Comment (ari-mercado) [inline Project-II-Calculator/calculator/src/components/ButtonComponents/ActionButton.js:34]: Similar to the mathSymbols array, a unique key should be used for each element in the list.
https://reactjs.org/docs/lists-and-keys.html

File: Project-II-Calculator/calculator/src/components/ButtonComponents/Button.css [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  background: white;
+  width: 75%;
+  display: flex;
+  flex-direction: column;
+}
+
+.numb-buttons {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap-reverse;
```
Comment (ari-mercado) [inline Project-II-Calculator/calculator/src/components/ButtonComponents/Button.css:11]: great use of wrap-reverse!

**Review by ari7946 — APPROVED** [review]
## Code Review Checklist

- [x] Follows Lambda Git Flow 
- [x] Frequent, clear, and descriptive commits
- [x] No errors in the console
- [x] Code is written by the student
- [ ] Code is sufficiently commented
- [x] Code is organized and easy to read
- [x] MVP Complete
- [ ] At least 1 stretch completed

### Notes:
Great work, Aaron! The calculator looks very nice. The components are well structured. The implementation of flexbox is brilliant. The only warning I noticed can be easily fixed with each element in the array having a unique *key*.

**Pull request merged**